com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1867,6 +1867,9 @@ AOVRequestData aovRequest
                         CustomPassVolume.Update(hdCamera);
                 }
 
+                if (m_CurrentDebugDisplaySettings != null && m_CurrentDebugDisplaySettings.data.UseDebugGlobalMipBiasOverride())
```

Comment:
It feels like this should probably be done in HDCamera.Update().
You can access the current debug with hdrp.m_CurrentDebugDisplaySettings (which should probably be renamed :p )

Reply:
Or maybe we can just update the global CB with a non zero Global Mip Bias only around passes where we do DrawRenderers calls? 
That's quite a few additional update to the global CB but might not be that bad.

Reply:
This has been fixed!